test/Routine.Test/Client/EqualityAndFormattingTest.cs

```diff
@@ -1,123 +1,123 @@
-
-        Assert.AreEqual(left, right);
-        Assert.AreNotSame(left, right);
-        Assert.AreNotEqual(left, other);
-
-        Assert.AreEqual(left.GetHashCode(), right.GetHashCode());
-        Assert.AreNotEqual(left.GetHashCode(), other.GetHashCode());
-    }
-
-    [Test]
-    public void Roperation_implements_formatting_and_equality_members()
-    {
-        ModelsAre(Model("model").Operation("operation"), Model("model2").Operation("operation2"));
-
-        var left = new Rapplication(_mockObjectService.Object)["model"].Operation["operation"];
-        var right = new Rapplication(_mockObjectService.Object)["model"].Operation["operation"];
-        var other = new Rapplication(_mockObjectService.Object)["model2"].Operation["operation2"];
-
-        Assert.AreEqual(left, right);
-        Assert.AreNotSame(left, right);
-        Assert.AreNotEqual(left, other);
-
-        Assert.AreEqual(left.GetHashCode(), right.GetHashCode());
-        Assert.AreNotEqual(left.GetHashCode(), other.GetHashCode());
-    }
-
-    [Test]
-    public void Rparameter_implements_formatting_and_equality_members()
-    {
-        ModelsAre(Model("model").Operation("operation", PModel("arg1")), Model("model2").Operation("operation2", PModel("arg1")));
-
-        var left = new Rapplication(_mockObjectService.Object)["model"].Operation["operation"].Parameter["arg1"];
-        var right = new Rapplication(_mockObjectService.Object)["model"].Operation["operation"].Parameter["arg1"];
-        var other = new Rapplication(_mockObjectService.Object)["model2"].Operation["operation2"].Parameter["arg1"];
-
-        Assert.AreEqual(left, right);
-        Assert.AreNotSame(left, right);
-        Assert.AreNotEqual(left, other);
-
-        Assert.AreEqual(left.GetHashCode(), right.GetHashCode());
-        Assert.AreNotEqual(left.GetHashCode(), other.GetHashCode());
-    }
-
-    [Test]
-    public void Robjects_implements_formatting_and_equality_members()
-    {
-        ObjectsAre(Object(Id("value", "model")), Object(Id("value2", "model")));
-
-        var left = Robj("value", "model");
-        var right = Robj("value", "model");
-        var other = Robj("value2", "model");
-
-        Assert.AreEqual(left, right);
-        Assert.AreNotSame(left, right);
-        Assert.AreNotEqual(left, other);
-
-        Assert.AreEqual(left.GetHashCode(), right.GetHashCode());
-        Assert.AreNotEqual(left.GetHashCode(), other.GetHashCode());
-    }
-}
+using Routine.Client;
+
+namespace Routine.Test.Client;
+
+[TestFixture]
+public class EqualityAndFormattingTest : ClientTestBase
+{
+    [Test]
+    public void Rapplication_implements_equality_members()
+    {
+        ModelsAre(Model("model"));
+
+        var left = new Rapplication(_mockObjectService.Object);
+        var right = new Rapplication(_mockObjectService.Object);
+
+        Assert.That(right, Is.EqualTo(left));
+        Assert.That(right, Is.Not.SameAs(left));

```

Comment:
left right olanlarda ikisinin esitligine bakiyoruz, expected actual durumu gibi degil yani. o nedenle bunlari `Assert.That(left, Is.EqualTo(right))` olarak duzenleyelim.